lib/aptible/api/vpn_tunnel_configuration.rb

```diff
@@ -40,11 +52,11 @@ def our_networks
       end
 
       def phase1_alg
-        "#{phase1_encryption_alg}-#{phase1_auth_alg};#{phase1_dh_group}"
+        "#{phase1_encryption_alg}-#{phase1_auth_alg};#{DH_KEYWORD[phase1_dh_group]}"
```

Comment:
Do we need to do any mapping for encryption and authentication algorithms or are they friendly enough?